### PR TITLE
Add description attribute to SearchableItem type

### PR DESCRIPTION
**Pull request by dblandin**
Pulls out the description included in the Elasticsearch result object.

### CODE REVIEW

**Comment by dblandin** [comment]
Merging this in!

**Comment by mzikherman** [comment]
Whoops, yup!

File: src/schema/searchableItem.ts [hunk 1]
```diff
@@ -36,6 +36,10 @@ export const SearchableItem = new GraphQLObjectType<any, ResolverContext>({
       type: new GraphQLNonNull(GraphQLID),
       resolve: item => toGlobalId("SearchableItem", item._id),
     },
+    description: {
+      type: GraphQLString,
+      resolve: item => item.description,
```
Comment (mzikherman) [inline src/schema/searchableItem.ts]: #minor this resolver is actually totally optional, since you're just destructuring a same-named key from the object.

Comment (dblandin) [reply src/schema/searchableItem.ts]: Ahh, good to know! Thanks!

**Pull request merged**